### DISCUSSION TITLE
Student Developer Pack Proof issue

### DISCUSSION THREAD

**Discussion by alejandraheredia** [community/community#31806] [GitHub Education] [answered]
Hi, I was trying to get my student benefits but the page doesn't let me add proof of my academic status. I've tried different browsers and I still have this problem, the page keeps refreshing and I can't add the image to process my application ... I'll be glad to get any help!!

**Comment by HimanshuSinghNegi** [comment, marked as answer]
Do not upload any image of your ID proof , just click 'take an image' and click image of your ID proof which has your current enrolling year mentioned, this method worked for me.

**Reply by Gogoal-L** [reply]
I have the same problem, I can't use my image or PDFs

**Reply by xGuiilherme** [reply]
How do I renew access with email educational for Student Developer Pack? I tried to renew using email but it didn't work.
![image](https://github.com/user-attachments/assets/6e6c978f-3b5b-4ac0-9054-8454ec98a4a6)

**Comment by xGuiilherme** [comment]
I have the same problem, I can't use my image for the receipt, help me please.

![image](https://github.com/user-attachments/assets/c05033e7-a7c9-40a5-abb6-0d5721a64da9)
